### ISSUE TITLE
Add example of how to create the database tables

### ISSUE THREAD

**Issue by kinware** [fantix/gino#74]
All of the current examples punt on showing how to to create the database tables:

```# You will need to create the database and table manually```

Requiring the user to create the database itself is ok, but having to create all application tables manually would, at least for me, be a show stopper for adopting Gino.

We would need to be able to use something equivalent to SQLAlchemy MetaData.sync_all(). I tried calling Gino().sync_all() (as Gino inherits from sa.MetaData), but it complained about not having a database connected to it. Maybe I did something wrong?

Does Gino() support a way to semi-automatically create the database tables from the application defined Models? This would be used either on application startup or from separate adminstration scripts. We might even defers to calling non-async methods of SQLAlchemy before the event loop is started. Anything that allows us to use the defined models would be good.

**Comment by fantix** [comment]
Thanks for commenting!

Yes it is one of the solutions to call non-async methods of SQLAlchemy before the event loop is started, like adding this in e.g. `examples/basic.py`:

```python
if __name__ == '__main__':
    import asyncio
    import sqlalchemy
    db.create_all(bind=sqlalchemy.create_engine('postgresql://localhost/gino'))

    loop = asyncio.get_event_loop()
    loop.run_until_complete(main())
```

While in production, alternative solution like [Alembic](https://github.com/alembic/alembic) is usually a better choice. Therefore `create_all` was not included in the examples to avoid misleading.

What you met with malfunctioning exposed `MetaData` API is an issue I think, GINO should either offer async version of them, or hide them to avoid confusion.

**Comment by kinware** [comment]
First: Thank you for your hard work on Gino! Enjoying learning async postgres :)

I created a PR based on the above comments.

https://github.com/fantix/gino/pull/75.

**Comment by fantix** [comment]
Good to know 😃  merged #75, shall track the issue in #76.
